src/ft2font_wrapper.cpp

```diff
@@ -534,6 +534,31 @@ PyFT2Font_set_size(PyFT2Font *self, double ptsize, double dpi)
     self->x->set_size(ptsize, dpi);
 }
 
+const char *PyFT2Font_set_features__doc__ = R"""(
+    Set the font feature tags used for the font.
+
+    Parameters
+    ----------
+    features : tuple[str, ...]
+        The font feature tags to use for the font.
+
+        Available font feature tags may be found at
+        https://learn.microsoft.com/en-us/typography/opentype/spec/featurelist
+)""";
+
+static void
+PyFT2Font_set_features(PyFT2Font *self, py::object features_obj)
+{
+    std::vector<std::string> features;
+    if (!features_obj.is_none()) {
+        auto features_list = py::cast<py::tuple>(features_obj);
+        for (auto &feature : features_list) {
+            features.push_back(feature.cast<std::string>());
+        }
+    }
+    self->x->set_features(std::move(features));
```

Comment:
Just to check my understanding, this is where we actually tell freetype to do anything ?